lib/src/html_generator.dart

```diff
@@ -5,9 +5,8 @@
 library dartdoc.html_generator;
 
 import 'dart:io';
-
+import 'dart:developer';
```

Comment:
Is there a dependency on a version of the sdk for this? If so, we should update our sdk dep.